Makefile

```diff
@@ -94,7 +94,8 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
+	! go tool vet --shadow . 2>&1 | grep -vE 'declaration of err shadows declaration at (proto/text.go:535|proto/text.go:570|protoc-gen-gogo/main.go:84)'
```

Comment:
> Whats the harm is just waiting until it is fixed?

It's probably going to be some time before that happens.

> The harm in committing this now is having the next committers possibly struggle around false positives and having to explain to them how to add their exceptions to the list.

Yeah, that's a risk. On the other hand, it can prevent bugs like this one: https://github.com/gogo/protobuf/pull/89/files#diff-9878bbf47ee5b9142c5e6fe30f61cbabR255